artichoke-backend/src/extn/core/kernel/require.rs

```diff
@@ -187,9 +187,6 @@ pub mod method {
         let files = if path.is_relative() {
             path = PathBuf::from(base);
             let mut files = Vec::with_capacity(2);
-            if !args.file.ends_with(".rb") {
-                files.push(path.join(format!("{}.rb", args.file)))
```

Comment:
non-blocking for merge: in the absence of this fallback, we can remove the `Vec`.